content/changelog/2024-05-31.md

```diff
@@ -0,0 +1,47 @@
+---
+description: Anonymize sensitive data with Neon and Neosync, join our Early Access Program, and more  
+---
+
+### Anonymize sensitive data with Neon and Neosync
+
+We are excited to announce that Neon has partnered with [Neosync](https://www.neosync.dev/) to provide developers with a complete solution for branching Postgres databases with anonymized data. Neosync is an open-source platform that helps developers anonymize production data and sync it across their environments for a better developer experience.
+
+<YoutubeIframe embedId="IcoOpnAcO1Y" />
+
+Both Neon and Neosync offer generous free tiers. If you’re interested in trying it out, follow the steps in our [Neosync guide](/docs/guides/neosync-anonymize) to get started.
+
+### Join the Neon Early Access Program from the Neon Console
+
+You can now join the **Neon Early Access Program** directly from the Neon Console to be among the first to try new Neon features. From your **Profile** menu, select **Account Settings** > **Early Access**, and submit your Discord username. 
+
+![Early Access Program console](/docs/relnotes/early_access_console.png)
+
+The benefits of joining include:
+
+- **Exclusive early access**: Get a first look at upcoming features before they go live.
+- **Private community**: Gain access to a dedicated Discord channel to connect with the Neon team and provide feedback to help shape what comes next.
+- **Weekly insights**: Receive updates on Neon's latest developments and future plans.
+
+### Computes, roles, and databases moved to branch pages in the Neon Console
+
+Computes, roles, and databases are now accessed from individual branch pages in the Neon Console instead of the Neon Console sidebar. Each of those objects is associated with a particular branch in a Neon project, and this change is intended to reflect that relationship.
+
+![new branches page](/docs/relnotes/new_branch_page_with_tabs.png)
+
+With this change, the compute endpoint delete option was moved to the **Edit compute endpoint** drawer, accessed by clicking **Edit** on the **Computes** tab.
```

Comment:
```suggestion
With this change, we also moved the compute endpoint delete option to the **Edit compute endpoint** drawer, which you can access by clicking **Edit** on the **Computes** tab.
```

Reply:
I removed this content, so I'll apply these suggestions to our stashed items for next week! Thank you!